mozilla-release/mobile/android/app/mobile.js

```diff
@@ -433,7 +433,7 @@ pref("dom.ipc.plugins.enabled", false);
 // The breakpad report server to link to in about:crashes
 pref("breakpad.reportURL", "https://crash-stats.mozilla.com/report/index/");
 
-pref("app.support.baseURL", "https://support.mozilla.org/1/mobile/%VERSION%/%OS%/%LOCALE%/");
+pref("app.support.baseURL", "https://www.ghostery.com/faqs/");
```

Comment:
And for Cliqz?

Reply:
I second Ravjit, please add support for Cliqz